planutils/__init__.py

```diff
@@ -40,8 +56,14 @@ def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument("-i", "--install",
-                        help="install an individual or collection of planners ('list' shows the options)",
-                        metavar="{planner or collection or list}")
+                        help="install an individual package such as a planner ('list' shows the options)",
+                        metavar="{package name}")
+    
+    parser.add_argument("-u", "--uninstall",
+                        help="uninstall a package, and any dependencies that are no longer required ('list' shows those that are installed)",
+                        metavar="{package name}")
+    
+    parser.add_argument("--check_installed", help="check if a package is installed")
```

Comment:
I think using a hyphen instead of an underscore would be the more common convention.

Reply:
Also, maybe these arguments all should be [subcommands](https://docs.python.org/2/library/argparse.html#sub-commands). This would be more in line with apt (`apt install foo` vs. `apt --install foo`) and would allow you to add different options to each command later on (e.g., an option of how to deal with dependencies when uninstalling).
Also, I'd turn `list` into a subcommand as well instead of having it as a parameter to `install`.